### ISSUE TITLE
White matter loss

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7262]
**Preferred term label:**
White matter loss.

**Synonyms**
White matter injury

**Definition (free text, please give PubMed ID)**
White matter loss refers to myelin deficit in the developing cerebrum (PMID: 34803612; 24961618; 12794739)	.

**Parent term (use hpo.jax.org/app)**
Abnormal cerebral white matter morphology HP:0002500

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
adding term as

Reduced cerebral white matter volume

An abnormally low volume of the white matter of the brain.

This feature can be ascertained by modalities including segmented volumetric magnetic resonance imaging. Reduced white matter volume may manifest as sulcal deepening and widening, enlarged ventricles, and an increased ratio of cerebrospinal fluid to gray and white matter.


Clinically it may be hard to know if reduce volume is related to loss or hypoplasia, it is best to keep the term general.
